uniffi_bindgen/src/bindings/swift/templates/RecordTemplate.swift

```diff
@@ -1,4 +1,4 @@
-public struct {{ rec.name()|class_name_swift }}:  ViaFfiUsingByteBuffer, ViaFfi, Equatable {
+public struct {{ rec.name()|class_name_swift }}:  ViaFfiUsingByteBuffer, ViaFfi, Equatable, Hashable {
```

Comment:
I don't object to it, but I'm curious why you needed to add `Hashable` here.

Reply:
No real need to do it here, this PR came out of some developer ergonomics when building an iOS app.